scylla-cdc-base/src/main/java/com/scylladb/cdc/model/worker/Consumer.java

```diff
@@ -39,10 +40,30 @@ public static Consumer forTaskAndRawChangeConsumer(TaskAndRawChangeConsumer c) {
         return new Consumer(c);
     }
 
+    public static Consumer syncTaskAndRawChangeConsumer(BiConsumer<Task, RawChange> c) {
+        return new Consumer(new TaskAndRawChangeConsumer() {
+            @Override
+            public CompletableFuture<Void> consume(Task task, RawChange change) {
+                c.accept(task, change);
+                return CompletableFuture.completedFuture(null);
+            }
+        });
+    }
+
```

Comment:
I'm not sure whether we want a blocking consumers to be allowed at all. Why do you need this at all? Is this used in your batching PR?

Reply:
No, it is convenience. I would say it is not for _blocking_ consumers, but rather ones that do not actually do anything in particular. I.e. demos doing println().